### DISCUSSION TITLE
Size of work surface

### DISCUSSION THREAD

**Discussion by Stuhli1** [arkypita/LaserGRBL#1697] [Q&A]
How do I change the size of the work surface?
I have 650 x 550 mm. What do I have to enter where?
It only shows me 320 x 160 and in the Config at $100 there ist 80....

Sorry for my english - i am from germany

**Comment by StuartB4** [comment]
Edit these 2 to the X and Y size of the machine.


![Untitled](https://user-images.githubusercontent.com/16980026/147897626-1d88fb6b-c561-44d4-b355-4327231093f3.png)
